AppController/djinn.rb

```diff
@@ -4953,6 +4941,13 @@ def check_running_apps()
     to_end = []
     APPS_LOCK.synchronize {
       @app_info_map.each { |app, info|
+        # TaskQueue servers needs to ensure that taskqueue server is
```

Comment:
This comment is a bit confusing because the difference between "Taskqueue server" and "taskqueue server" is not clear. I would recommend: "Machines with a taskqueue role need to ensure that the latest queue configuration files are loaded."